bin/make_dht_configs

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import json
+import os
+
+from syncr_backend.constants import DEFAULT_DPS_CONFIG_FILE
+from syncr_backend.constants import DEFAULT_PKS_CONFIG_FILE
+from syncr_backend.init.node_init import get_full_init_directory
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "listenport",
+        type=str,
+        help='Port to listen on for DHT',
+    )
+    parser.add_argument(
+        "--bootstrap-ips",
+        type=str,
+        help='IP of the DHT bootstrap',
+    )
+    parser.add_argument(
+        "--bootstrap-ports",
+        type=int,
+        help='Port of the DHT bootstrap',
+    )
+    args = parser.parse_args()
+
+    iplist = args.bootstrap_ips.split(',')
+    portlist = list(map(lambda x: int(x), args.bootstrap_ports.split(',')))
+    if len(iplist) != len(portlist) or len(iplist) == 0:
+        raise Exception("Must have at least 1 bootstrap ip")
```

Comment:
Instead of raising an exception, print that to stderr (or just stdout via `print`), and do `exit(1)`